app/graph/types/source_type.rb

```diff
@@ -12,7 +12,6 @@
   field :permissions, types.String
   field :pusher_channel, types.String
   field :lock_version, types.Int
-  field :medias_count, types.Int
```

Comment:
I think we still want to count the medias published by a source.

Reply:
Ok will keep it